app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -159,12 +159,29 @@ def queue_url_to_arn(queue_url)
   # @param [Aws::SQS::Types::Message] message
   def parse_event(message)
     event = JSON.parse(JSON.parse(message.body)['Message'])
-    $log.info("#{log_header} Found SNS Message with message type #{event["messageType"]}")
-    return unless event["messageType"] == "ConfigurationItemChangeNotification"
+
+    if event["messageType"] == "ConfigurationItemChangeNotification"
+      # Aws Config Events
+      event["eventType"]   = parse_event_type(event)
+      event[:event_source] = :config
```

Comment:
@bronaghs my understanding is that we are getting AWS Config events along side the others that are used in the `elsif` parts below.

The check for `AwsApiCall` (CloudTrail) is because currently EBS events don't work (even though you configure CW to publish EBS-related events, they are never delivered over SNS; @Ladas mentioned he needs to check with AWS support). CloudTrail supposedly pushes all AWS API calls, including those that occur when you are creating, deleting, ... volumes/snapshots.

Last `elsif` checks for `EC2 Instance State-change Notification` which is a CW event, specific to EC2 instances.